src/plugins/discover/public/application/components/chart/download_csv_button/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
i think wrong license header

Reply:
oops i will update. still not 100% sure what the correct header is, is this it?

```
/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 *
 * Any modifications Copyright OpenSearch Contributors. See
 * GitHub history for details.
 */
```

Reply:
Let es lint auto apply that. If your editor is setup correctly, it should happen automatically

Reply:
Nit: Also we should make this a CI step